src/config.ts

```diff
@@ -37,14 +41,15 @@ export async function getConfig(): Promise<Config> {
         backoffFactor *= 2;
       }
       attemptCount += 1;
-      const resp = await got(`https://api.solend.fi/v1/config?deployment=${getApp()}`, { json: true });
-      return resp.body as Config;
+      const resp = await got(`https://api.solend.fi/v1/markets/configs?deployment=${getApp()}`, { json: true });
```

Comment:
```suggestion
      const resp = await got(`https://api.solend.fi/v1/markets/configs?scope=solend&deployment=${getApp()}`, { json: true });
```
should add the scope `solend` for safety so